src/components/NavigationPills/NavigationPills.js

```diff
@@ -70,8 +70,7 @@ class NavigationPills extends React.Component {
         return
       }
 
-      // eslint-disable-next-line jsx-a11y/label-has-associated-control
-      return <label style={styles.labelStyles}>{label}</label>
```

Comment:
Instead of always using h2, we should probably make that a prop that can be passed in.

Reply:
Yeah I was thinking of adding something similar to the Text component's `elementType`. Here I might make it `labelElementType`.

I checked and this component is only used in two similar places in instacart (at the top of a retailer chooser). Do you think it's "worth it" to add this level of configurability rather than just making it an h2 and calling it a day?

Reply:
I think it's worth the effort honestly. It also means you could make this change without impacting the other code by making the default be label.

Reply:
Cool. Will update.